lib/samson/clair.rb

```diff
@@ -12,6 +12,7 @@ class << self
       def append_job_with_scan(job, docker_tag)
         return unless clair = ENV['HYPERCLAIR_PATH']
 
+        ActiveRecord::Base.clear_active_connections!
```

Comment:
this is to avoid the bug, at this point we don't know if there is a bug ... same code as https://github.com/zendesk/samson/blob/master/app/models/job_execution.rb#L43-L45
afaik have to do this before using db in a new thread